website/docs/docs/dbt-cloud/on-premises/faqs.md

```diff
@@ -64,3 +64,24 @@ spec:
 ```
 
 For more examples of using the nginx ingress controller, see [the Examples section](https://kubernetes.github.io/ingress-nginx/examples/tls-termination/) of their documentation.
+
+## Troubleshooting
+
+### Restarting the dbt or kotsadm Applications
+
+Certain tasks may require restarting the dbt or the kotsadm applications. In order to accomplish this, the below commands can be run.
+
+**Restarting dbt**
+
+```bash
+kubectl rollout restart deployment/api-gateway
+kubectl rollout restart deployment/app
+kubectl rollout restart deployment/scheduler
+```
+
+**Restarting kotsadm**
```

Comment:
we generally refer to `kotsadm` as the "configuration console", can you update this to indicate that? e.g.

"Restarting the Configuration Console (kotsadm)"

for each of these, you should indicate what the impact of running these commands will be. for kotsadm, the config console will be unavailable for a few minutes. for dbt, the actual app will be unavailable for a few minutes.